Dockerfile

```
@@ -4,34 +4,95 @@ FROM python:3.10.1-alpine
 
 ARG VERSION
 
+###
 # For a list of pre-defined annotation keys and value types see:
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
+#
 # Note: Additional labels are added by the build workflow.
+###
 LABEL org.opencontainers.image.authors="mark.feldhousen@cisa.dhs.gov"
```

Not to rain on @felddy 's parade but should we make this consistent with our [Python packages](https://github.com/cisagov/skeleton-python-library/blob/7dc735b4ad8d66f0794dd6d3e16bae869bb9958b/setup.py#L59-L61)?

```suggestion
LABEL org.opencontainers.image.authors="github@cisa.dhs.gov"
```

---

I agree that `github@cisa.dhs.gov` makes sense from a `cisagov`-wide perspective, but what if we change this email address to `vm-fusion-dev-group@trio.dhs.gov` in repos maintained by @cisagov/team-ois?

---

@jsf9k - I like your suggestion of using `vm-fusion-dev-group@trio.dhs.gov` for our repos.

---

I'm fine with either. We can also use both as that label is a freeform string so

```dockerfile
LABEL org.opencontainers.image.authors="github@cisa.dhs.gov, vm-fusion-dev-group@trio.dhs.gov"
```
or similar could be used.

---

OK, let's make it a practice going forward to add the `vm-fusion-dev-group@trio.dhs.gov` email address to the `org.opencontainers.image.authors` label for repos that we (@cisagov/team-ois) maintain.

I'll add a blurb to the `README` right now.